src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,37 @@
+import "./Introduce.scss";
+import { Link, Navigate, UseNavigate, useNavigate } from "react-router-dom";
+
+const Introduce = () => {
+  const navigate = useNavigate();
+  const goToProduct = () => {
+    navigate("/product");
+  };
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+  const handleButton = () => {
+    console.log("button 태그 동작");
+  };
```

Comment:
```suggestion
  const navigate = useNavigate();
  
  const handleInput = () => {
    console.log("input 태그 동작");
  };
  
  const handleButton = () => {
    console.log("button 태그 동작");
        navigate("/product");
  };
```

함수끼리 가독성을 위해 한칸씩 띄워주세요!
그리고 버튼에 적용되는 함수를 하나로 구성해주세요!